src/index.js

```diff
@@ -12,7 +12,7 @@ import { setToken } from './middlewares/auth';
 
 const store = createStore(rootReducer, compose(
     applyMiddleware(thunk, setToken),
-    // window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Please comment this out when you submit PRs

Reply:
great, i will